src/ISO3166.php

```diff
@@ -137,12 +152,12 @@ public function getIterator()
     private function getBy($key, $value)
     {
         foreach ($this as $country) {
-            if (0 === strcasecmp($value, $country[$key])) {
+            if ($value === $country[$key]) {
                 return $country;
             }
         }
 
-        throw new \OutOfBoundsException('ISO 3166-1 does not contain: ' . $value);
+        throw new \OutOfBoundsException(sprintf('ISO 3166-1 does not contain: %s', $value));
```

Comment:
>  The message needs to be re-formatted to say that the value is not contain in the collection

But that is that the message currently says?

Or do you mean that "ISO3166-1" is too explicit, and you think we should find a more neutral term to indicate that the "current data set" does not contain data for the given identifier?